Sources/Datadog/Datadog.swift

```diff
@@ -246,6 +256,9 @@ public class Datadog {
             userInfoProvider: userInfoProvider,
             launchTimeProvider: launchTimeProvider
         )
+
+        // After everything is set up, send crash report if available
+        CrashReportingFeature.instance?.sendCrashReportIfFound()
```

Comment:
Assuming the implementation is synchronous, could this become a bottleneck? And even more so if it's called when the App delegate is called back on the main thread.

Reply:
Yes, if kept in the current form it's definitely a bottleneck as crash report deserialisation (in `PLCrashReporter`) may take a while. Both `sendLog()` and `sendRUMEvent()` will be fast in our code, as they are dispatched to background thread by the `Writer`. However, as explained in [the previous comment](https://github.com/DataDog/dd-sdk-ios/pull/397/files/9901f739f1207f3a44c1d0fc3c258c9485802f11#r569217837) the `readPendingCrashReport(completion:)` will be executed off the main thread in next PR, so the impact will be reduced to just  `queue.async {}`.